annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -112,7 +121,7 @@ public AnnotatedRecord annotateRecord(MutationRecord record, boolean replaceHugo
 
         //check if record already is annotated
         Map<String, String> additionalProperties = mRecord.getAdditionalProperties();
-        if(additionalProperties.containsKey("HGVSp_Short") && !reannotate && !additionalProperties.get("HGVSp_Short").isEmpty()) {
+        if(additionalProperties.containsKey("HGVSp_Short") && !reannotate && !additionalProperties.get("HGVSp_Short").isEmpty() || hgvspNullClassifications.contains(mRecord.getVariant_Classification())) {
```

Comment:
This doesn't change the logic, but I think it reads better if you have:

if (additionalProperties.containsKey("HGVSp_Short") && !additionalProperties.get("HGVSp_Short").isEmpty() && !reannotate...